### DISCUSSION TITLE
Just a Question about maybe having like a server

### DISCUSSION THREAD

**Discussion by QMand** [loriswit/rlcm#2] [Ideas]
Hi there, 
This mod is awesome and needs more publicity now that servers was removed a while ago.
I'm wondering if this can be used as a sort of foundation on getting this game 'online' again with a hosted server.

**Comment by loriswit** [comment]
I'm not exactly sure what you mean by a "hosted server".

- Either we write a server app from scratch that behaves exactly like the UbiSoft servers used to, and make some changes to the game files in order to communicate we our own server. This would surely require some heavy reverse engineering on the game, and would probably be very hard to achieve.

- Or we make a custom, simpler app that runs on a server, which simply feeds a new seed every day to this challenge manager, and maybe also stores results and ghosts from players to generate a leaderboard. This would definitely be a lot easier, but would be less transparent to the players (they would still need to install the training room, jump into the correct painting, select the correct game mode).

Either way, I personally don't have the time to design and develop any of these right now, but since this is an open source project, feel free to submit a PR with your own implementation. I'm always available if you need help.
